cli/azd/pkg/tools/pack/pack.go

```diff
@@ -165,6 +165,17 @@ func (cli *packCli) version(ctx context.Context) (semver.Version, error) {
 	return version, nil
 }
 
+func (cli *packCli) enableExperimental(ctx context.Context) error {
```

Comment:
Changes here are a result of switching the default builder image. We'll need to enable experimental features for `pack` to work.